src/omega_edit/client.ts

```diff
@@ -145,7 +152,7 @@ export function activate(ctx: vscode.ExtensionContext) {
         port: number | undefined = undefined,
         fileToEdit: string = ''
       ) => {
-        await getOmegaEditPort(port)
+        await getOmegaEditPort(serverPort)
```

Comment:
See previous comment to remove the port param from the `getOmegaEditPort` function since its not needed anymore.